File: Java Programmes/College Assignment/Assignment 1/PrimeCheck.java
import java.util.*;
class PrimeCheck 
{
    public static void main(String args[]) 
    {
        int i,num;
        System.out.print("Enter the Number : ");
        Scanner in = new Scanner(System.in);
        num=in.nextInt();
        for(i=2;i<num;i++)
        {
            if(num%i==0)
            {
                System.out.println(num+" is not Prime Number .");
                break;
            }
        }
        if(i==num)
            System.out.println(num+" is Prime Number .");    
    }    
}

File: Python_Programs/ML Assignment/Assignment 2/prime_check.py
#Program For Checking the Number is Prime Or Not
num_str=input("Enter the number : ")
num=int(num_str)
if(num==1):
    print(num,"is Not a Prime Number")
if(num==2 or num==3):
    print(num,"is a Prime Number !")
elif num>2:
    for i in range(2, (num//2)+1):
        if (num % i) == 0:
            break
    if(i==(num//2)):
        print(num,"is a prime Number ..")
    else:
        print(num, "is not a Prime Number !")